### PR TITLE
Feat/cypress test home page

### PR DESCRIPTION
**Pull request by raz-joan**
What (if any) features are you implementing?
- Installed cypress
- Wrote first test for what elements should be there on the home page

What (if anything) did you refactor?
- n/a
Were there any issues that arose?
- n/a
Is there anything that you need from your teammate?
- n/a
Any other comments, questions, or concerns?
- Per y'alls suggestion, let's create two describe blocks for each user view. One for what elements should be there upon load and another to test for user interaction.

### CODE REVIEW

**Review by michelecomfort — COMMENTED** [review]
Cypress tests for home page look great!

Way to handle those merge conflicts on our rebase!

**Review by colganmeanor — COMMENTED** [review]
Quite a journey to get all this code updated here. Thank you so much for helping to facilitate this @raz-joan and @michelecomfort. 

FYI we reviewed all of this code together while this PR was being opened.

File: cypress/integration/homepage.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        cy.get('[data-cy=header')
+            .should('exist')
+    })
+
+    it('should have a title', () => {
+        cy.get('h1')
+            .contains('Board Game Bartender')
+    })
+
+    it('should have a PairingForm component', () => {
```
Comment (michelecomfort) [inline cypress/integration/homepage.js:16]: Great use of data-cy labels!

**Review by michelecomfort — COMMENTED** [review]
Nice teamwork

**Pull request merged**